plugins/toolkit/jetbrains-ultimate/tst/software/aws/toolkits/jetbrains/services/rds/auth/IamAuthTest.kt

```diff
@@ -168,7 +208,8 @@ class IamAuthTest {
         hasCredentials: Boolean = true,
         hasBadHost: Boolean = false,
         hasSslConfig: Boolean = true,
-        dbmsType: Dbms = Dbms.POSTGRES
+        dbmsType: Dbms = Dbms.POSTGRES,
+        credentialId: String = this.credentialId
```

Comment:
new parameter is not actually used anywhere

Reply:
it is being used on line number 227. It is used to store the profile name.